node-commons/src/main/java/org/apache/iotdb/commons/partition/GroupType.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-package org.apache.iotdb.consensus.common;
+package org.apache.iotdb.commons.partition;
```

Comment:
Place the GroupType in org.apache.iotdb.commons.consensus might be better.

Reply:
Done